extensions/blocks/podcast-player/components/playlist.js

```diff
@@ -8,7 +8,31 @@ import { __ } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
-import { STATE_ERROR } from '../constants';
+import * as episodeIcons from '../icons/episode-icons';
+import { STATE_ERROR, STATE_PLAYING } from '../constants';
+
+const TrackIcon = ( { isPlaying, isError, className } ) => {
+	let name;
+
+	if ( isError ) {
+		name = 'error';
+	} else if ( isPlaying ) {
+		name = 'playing';
+	}
+
+	const icon = episodeIcons[ name ];
+
+	if ( ! icon ) {
+		// Return empty element - we need it for layout purposes.
+		return <span className={ className } aria-hidden="true" />;
+	}
+
+	return (
+		<span className={ `${ className } ${ className }--${ name }` } aria-hidden="false">
+			{ icon }
```

Comment:
We can insert the previous icon text as a `<VisuallyHidden>` text here, since the `icon` has `aria-hidden` hardcoded as `true`. So, `<VisuallyHidden>Playing</VisuallyHidden>`, etc.

Reply:
`VisuallyHidden` is a Gutenberg component. Bringing it in would likely mean loading all Gutenberg components on the frontend too. Let's refrain from it, unless 100% necessary.

Reply:
Ah, we can just do a `<span className="whatever-the-visually-hidden-class-floating-around-is">`. If there isn't a global one we can add it to our block. https://a11yproject.com/posts/how-to-hide-content/

Reply:
See https://github.com/Automattic/jetpack/pull/15101#discussion_r398618668